integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/sdx/SdxRepairTests.java

```diff
@@ -153,6 +155,7 @@ public void testSDXMediumDutyRepair(TestContext testContext) {
                     return testDto;
                 })
                 .awaitForHostGroups(List.of("gateway", "idbroker"), InstanceStatus.DELETED_ON_PROVIDER_SIDE)
+                .await(SdxClusterStatusResponse.DELETED_ON_PROVIDER_SIDE)
```

Comment:
I think this is not necessary here, because of we already waiting for the `DELETED_ON_PROVIDER_SIDE` in the previous line with the help of `awaitForHostGroups`

Reply:
You are right, removed it. Thanks!